test/mocks/MockAppConfig.scala

```diff
@@ -30,6 +30,11 @@ trait MockAppConfig extends MockFactory {this: TestSuite =>
       .expects()
       .returning(response)
 
+  def mockEmploymentUrl(response: String): CallHandler0[String] =
```

Comment:
should this be `mockEmploymentBaseUrl` to match the other url's listed in this file